### DISCUSSION TITLE
Is the header row available in Sylvan.Data.Csv?

### DISCUSSION THREAD

**Discussion by lc9er** [MarkPflug/Sylvan#152] [General]
Is the header row available? I'm working on a project where the csv headers are not known ahead of time, and some headers can be excluded at runtime. Is the header row captured anywhere? I tried using the first row with the Read() method, but that gave me the first row of data.

**Comment by MarkPflug** [comment]
You can call FieldCount and GetName immediately after calling Create to determine the column headers.

**Reply by lc9er** [reply]
Worked like a charm! Thank you for the help.
